DataParsers/XmlWeatherDataParsingStrategy.cs

```diff
@@ -0,0 +1,19 @@
+using System.Xml.Linq;
+using Models;
+namespace DataParsers
+{
+    public class XmlWeatherDataParsingStrategy : IWeatherDataParsingStrategy
+    {
+        public WeatherData ParseData(string dataFile)
+        {
+            string configXml = File.ReadAllText(dataFile);
+            XElement xml = XElement.Parse(configXml);
+            return new WeatherData
+            {
+                Location = xml.Element("Location").Value,
```

Comment:
Let's move it to a constant in the same class or use Enums , but keep in mind  the trade off of each one.
using Enum for the both parsers would suppose that both file formats  always have the same fields.